### ISSUE TITLE
커널 컴파일 중 clock skewed error.

### ISSUE THREAD

**Issue by yoona3316** [hyojeonglee/osfall2019#14]
./build- 로 컴파일하다가 어느순간 clock skewed error라는 워닝?에러?가 뜨는데,
시스템 시간과 소스파일 시간이 맞지 않기 떄문에 발생한다고 뜨네요..?
컴파일이 안되는 건 아닌거 같긴 한데;
인터넷으로 찾은 자료로 해결하려고 하니 해결이 제대로 되지 않아서요;
그냥 무시해도 되나요?
아래는 제가 참고한 자료입니다.
https://knoow.tistory.com/134

**Comment by hyojeonglee** [comment]
네 문제 원인은 말씀하신대로 make로 새로운 오브젝트 파일 생성 시 기존 오브젝트 파일의 시간과 현재 시간이 맞지 않아 발생합니다.
```
$ find . -exec touch {} \;
$ make clean
```
파일의 타임스탬프 변경 및 기존의 오브젝트 파일을 지워주는 위 명령어들 수행한뒤 다시 빌드 스크립트 돌려보시겠어요?

**Comment by yoona3316** [comment]
find . -exec touch {} \ 명령실행이 원래 오래 걸리는 작업인가요?

**Comment by yoona3316** [comment]
해결됐습니다. 감사합니다.!

**Comment by hyojeonglee** [comment]
같은 문제 발생시 다른 학생분들 참고를 위해 다시 오픈해두겠습니다~
